doc/examples/puppeteer/axe-puppeteer.js

```diff
@@ -27,7 +27,7 @@ const main = async url => {
 		// Inject and run axe-core
 		const handle = await page.evaluateHandle(`
 			// Inject axe source code
-			${axeCore.source}
+			var axe = ${axeCore.source}
```

Comment:
Why is this necessary? I don't think I understand what's happening here.

Reply:
Puppeteer loads [CommonsJS by default](https://github.com/puppeteer/puppeteer/issues/3685#issuecomment-453229607) so there is no global object after the source runs.

Reply:
This is not true. The comment you linked to is discussing something completely different.

Within the `Page#evaluateHandle` context, there is no `module` defined. Here's a script proving this:

```js
const puppeteer = require('puppeteer')

const test = async () => {
  const browser = await puppeteer.launch()
  const page = await browser.newPage()

  const handle = await page.evaluateHandle(`
    Promise.resolve({
      module: typeof module,
      define: typeof define,
      window: typeof window
    })
  `)

  const types = await handle.jsonValue()
  console.log('typeof module: "%s"', types.module)
  console.log('typeof define: "%s"', types.define)
  console.log('typeof window: "%s"', types.window)

  await page.close()
  await browser.close()
}

test().catch(err => {
  console.error(err)
  process.exit(1)
})
```

This will output:

```
typeof module: "undefined"
typeof define: "undefined"
typeof window: "object"
```

I do not understand why we need to add `var axe = ` in front of `axe.source`, and believe it's due to a bug.